<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>991</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(1, 145, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Selamat Datang!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#45ce36;&quot;&gt;Sofware SI OLEH-OLEH&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;DASHBOARD ADMIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>110</y>
      <width>1021</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 56, 59);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Nama Oleh-Oleh&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Harga&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Stok&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="namaBarang">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hargaBarang">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>301</width>
      <height>301</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 56, 59); color:white;
</string>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Nama Oleh-Oleh</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Harga</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jumlah</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="stokBarang">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="urutkan">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
    <property name="text">
     <string>Urutkan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tambahkan">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
    <property name="text">
     <string>Tambahkan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hapus">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;  border-radius:5px;  color:black;
</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simpan">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:5px; background-color:white; color:black;
</string>
    </property>
    <property name="text">
     <string>Simpan</string>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>namaBarang</zorder>
   <zorder>hargaBarang</zorder>
   <zorder>tableWidget</zorder>
   <zorder>stokBarang</zorder>
   <zorder>urutkan</zorder>
   <zorder>tambahkan</zorder>
   <zorder>hapus</zorder>
   <zorder>logout</zorder>
   <zorder>edit</zorder>
   <zorder>simpan</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
